schemas/BaseElements.xsd

```diff
@@ -2709,6 +2709,11 @@ http://www.epa.gov/compliance/monitoring/programs/caa/woodheaters.html</xs:docum
 			<xs:extension base="HVACSystemInfo">
 				<xs:sequence minOccurs="0">
 					<xs:element name="HeatPumpType" type="HeatPumpType" minOccurs="0"/>
+					<xs:element minOccurs="0" name="ColdClimate" type="xs:boolean">
+						<xs:annotation>
+							<xs:documentation>Is this a cold climate heat pump?</xs:documentation>
```

Comment:
Agreed. This could even have two levels. Variable Speed/Inverter technology keeps the systems running well at temps below 32ºF. As far as I know Mitsubishi is the only brand that runs at full capacity below 5ºF. But it’s only on their Hyper Heat models.

> On Nov 30, 2015, at 3:03 PM, GamalielL <notifications@github.com> wrote:
> 
> In schemas/BaseElements.xsd <https://github.com/hpxmlwg/hpxml/pull/58#discussion_r46195339>:
> 
> > @@ -2709,6 +2709,11 @@ http://www.epa.gov/compliance/monitoring/programs/caa/woodheaters.html</xs:docum
> >  			<xs:extension base="HVACSystemInfo">
> >  				<xs:sequence minOccurs="0">
> >  					<xs:element name="HeatPumpType" type="HeatPumpType" minOccurs="0"/>
> > +					<xs:element minOccurs="0" name="ColdClimate" type="xs:boolean">
> > +						<xs:annotation>
> > +							<xs:documentation>Is this a cold climate heat pump?</xs:documentation>
> My understanding is that "cold climate" heat pumps have inverter/variable speed drives and that is what gives them curves that are better in cold climates. Maybe we should add a boolean for that as a specific technical specification instead of the subjective "cold climate" designation.
> … <x-msg://55/#>
> —
> Reply to this email directly or view it on GitHub.
> 
>  <https://github.com/hpxmlwg/hpxml/pull/58#discussion_r46182204> <http://www.epa.gov/compliance/monitoring/programs/caa/woodheaters.html%3C/xs:docum> <https://github.com/nmerket> <https://github.com/nmerket> <https://github.com/gmdickey> <https://github.com/gmdickey> <https://github.com/hpxmlwg/hpxml/pull/58/files#r46182204>
> —
> Reply to this email directly or view it on GitHub <https://github.com/hpxmlwg/hpxml/pull/58/files#r46195339>.
> 


--------------------------------
Adam Stenftenagel
Co-Founder / CEO
Snugg Home
PO BOX 82
Boulder, CO
720-663-7836 x101
720-222-8604 fax
www.SnuggHome.com <http://www.snugghome.com/>
--------------------------------